src/PrestaShopBundle/Form/Admin/Sell/Product/Category/CategoriesType.php

```diff
@@ -41,11 +41,12 @@ class CategoriesType extends TranslatorAwareType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('product_categories', CategoriesCollectionType::class)
-            ->add('create_category', UnavailableType::class, [
-                'label' => $this->trans('Create a new category', 'Admin.Catalog.Feature'),
-                'label_tag_name' => 'h2',
-                'label_help_box' => $this->trans('If you want to quickly create a new category, you can do it here. Don’t forget to then go to the Categories page to fill in the needed details (description, image, etc.). A new category will not automatically appear in your shop\'s menu, please read the Help about it.', 'Admin.Catalog.Help'),
+            ->add('product_categories', CategoryTagsCollectionType::class)
+            ->add('add_categories_btn', ButtonType::class, [
+                'label' => $this->trans('Add categories', 'Admin.Catalog.Feature'),
+                'attr' => [
+                    'class' => 'add-categories-btn btn-outline-primary',
+                ],
```

Comment:
You should add a `default_category` `ChoiceType` select field here to select the default category

Reply:
We did decide (with @NeOMakinG & @matks) have no form type and leave it purely for frontend, as the form options should still be populated dynamically based on selected categories and ChoiceType doesn't have anything like collection's prototype :thinking:

Reply:
We did decide (with @NeOMakinG & @matks) have no form type and leave it purely for frontend, as the form options should still be populated dynamically based on selected categories and ChoiceType doesn't have anything like collection's prototype :thinking:

Reply:
solved with form listener